cl/assets/templates/cotton/data_table.html

```diff
@@ -20,7 +28,15 @@
     {% for row in rows %}
       <tr class="border-b-greyscale-200 border-b">
         {% for column in columns %}
-          <td class="text-start align-top text-sm font-normal p-2">
+          <td class="align-top text-sm font-normal p-2
+            {% if align == "data-center" and not forloop.first %}
+              text-center
```

Comment:
Same as the previous suggestion:

```suggestion
            {% if align == "data-center" %}
              text-center first:text-start
```